api/v2/checluster_types.go

```diff
@@ -85,6 +85,10 @@ type CheClusterDevEnvironments struct {
 	// +optional
 	// +kubebuilder:default:={pvcStrategy: per-user}
 	Storage WorkspaceStorage `json:"storage,omitempty"`
+	// PersistUserHome defines configuration options for persisting the `/home/user/`
```

Comment:
Can we not to mention specifically  `/home/user`?
For instance: `Defined configuration options for persisting a home user directory in a workspace`

Reply:
That's okay with me, will make the appropriate change 👍

Reply:
I did not modify [the documentation](https://github.com/eclipse-che/che-operator/pull/1737/files#diff-10f5235ef9752eb68e9938878907ee064b366dd04540ac315d0b8912860a38dcR445) for `persistUserHome.enabled` to remove the mention of `/home/user/` as it felt appropriate to explicitly state which directory would persist. However, I am open to reconsidering this decision.